app/models/zoom_meeting.rb

```diff
@@ -56,6 +60,14 @@ def self.personal_meeting_error
     InvalidMeetingError.new("It looks like that Zoom link is for a Personal Meeting Room. You will need to use a unique meeting instead.")
   end
 
+  def self.invalid_zoom_url_error
+    InvalidMeetingError.new("The Zoom URL entered is not valid. Please enter a URL that follows this format: 'https://turingschool.zoom.us/j/12345678901'.")
+  end
+
+  def self.invalid_meeting_id_length_zoom_url_error
+    InvalidMeetingError.new("The Zoom URL entered is not valid. Check that the meeting ID has 10-11 digits.")
+  end
```

Comment:
Per my feedback above, I don't think you need to get this specific with the error messages. You can reuse the invalid_error instead